public/sw.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+const VERSION = 1
+const CACHE = 'crime-data-explorer'
+
+self.addEventListener('install', event => {
```

Comment:
For service workers, I believe you have to use `self` since it is in a different thread that doesn't have a `window`